Project2015To2017Tests/UnsupportedProjectTypesTest.cs

```diff
@@ -4,6 +4,8 @@
 using System.Xml.Linq;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Project2015To2017;
+using Project2015To2017.Definition;
+using static Project2015To2017.Definition.Project;
```

Comment:
this line seems duplicate?

Reply:
Actually my VS is down ( pending repairs :) ), so this PR was coupled in Sublime Text and `dotnet build`. But actually I don't know if `using static` implies reference of outer namespace.

Reply:
No, that line is necessary for compilation (just checked it).